### PR TITLE
Sync from ModelDB zipfile.

### PR DESCRIPTION
**Pull request by olupton**
Update GitHub with changes found via the ModelDB API.

@ramcdougal I am not sure how you want to handle this.

### CODE REVIEW

File: batch_.hoc [hunk 1]
```diff
@@ -10242,7 +10242,7 @@ time("smap()")
 // @DPDPAM = (80,180)
 // @DPDPNM = (320)
 
-if (name_declared("dstr")) batch_flag=1 else batch_flag=0
+if (name_declared("dstr")) batch_flag=1 else batch_flag=1
```
Comment (ramcdougal) [inline batch_.hoc:10245]: the original made a lot more sense to me

Comment (pramodk) [reply batch_.hoc:10245]: thanks @ramcdougal for noticing this!

Seems like this is unintended change due to how we download and change the model while running it via nrn-modeldb-ci: https://github.com/neuronsimulator/nrn-modeldb-ci/blob/493e77d78f8d76356114b2a9515d76916062f1bf/modeldb/modeldb-run.yaml#L676

Now I am curious about other `sed & replace` thing that are being done in the same `modeldb-run.yaml`! 🤔.
Did you notice other such unexpected changes in the MOD file? May be we can relook into this again. cc: @olupton @alexsavulescu

**Comment by pramodk** [comment]
this should be closed.

File: batch_.hoc (hunk 1 above)
Comment (olupton) [reply batch_.hoc:10245]: I think @pramodk's explanation here is correct, and I see that he's reverted the change in #2 so I think we're good to go 🚀

**Pull request closed without merging**